x-pack/docs/en/security/backup-security-configuration.asciidoc

```diff
@@ -0,0 +1,263 @@
+the data which is *not* part of an ordinary configuration backup. This data
+describes:
+
+* the definition of users in the native realm (including hashed passwords)
+* role definitions
+* role mappings
+* application privileges
+* API keys
+
+Be advised that the index format is only compatible over a single major version.
+
+TIP: The `elasticsearch.yml` and `elasticsearch.keystore` files and the global
+cluster metadata contain settings for all the {stack} features. The
+`.security` index and all the other
+<<security-files,security configuration files>> contain configuration data
+exclusively for the {security-features}. It is easiest to backup and restore all
+the configuration information as a whole, rather than trying to cherry-pick only
+the security details.
+
+[float]
+[[backup-security-files]]
+==== Backing up configuration files
+
+The simplest way to backup configuration files is to copy the `$ES_PATH_CONF`
+directory. Overall, this procedure is identical to a conventional backup.
+
+NOTE: In a typical {es} deployment, all the configuration files are identical on
+each node.
+
+It is recommended that you back up the configuration files every time they are
+modified. This is not as difficult as it sounds because, in a typical
+deployment, after you achieve a desired configuration, you update these files
+infrequently. Configuration files purposefully contain only the settings that
+change rarely or are required for cluster formation or require file system
+privileges.
+
+TIP: For the YAML text files, you also have the option of storing diffs against a
+baseline or checking them into `git`. This method makes it easier for the {es}
+administrator to review the changes.
+
+The binary and encoded files, such as the private keys and the secure store
+file, are extra sensitive; they require restricted read access.
+
+[float]
+[[backup-security-snapshots]]
+==== Snapshotting the security index and the global cluster metadata
+
+To back up API-based configuration, which is stored inside the cluster, you use
+the <<modules-snapshots,regular snapshot functionality>>. The snapshot API
+saves the settings, the type mapping, and the contents of any index to a
+location on a cloud storage service or on a shared network file system. This is
+used to snapshot the `.security` index. The snapshot operation can also
+optionally capture the persistent settings from the global cluster metadata.
+
+. Create a repository that you can use to backup the `.security` index.
+Ideally this repository is dedicated only to this scope.
++
+--
+[source,js]
+-----------------------------------
+PUT /_snapshot/my_backup
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+-----------------------------------
+// CONSOLE
+// TESTSETUP
+
+You must have the elevated `manage` cluster privilege to create it, yet it's a
+relatively infrequent operation. The elevated privileges are required to prevent
```

Comment:
It's my _personal preference_ to leave the `yet ...` commentary out.  Just call it like it is.

> Creating a snapshot repository requires the `manage` cluster privilege.